### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by alexjhawk**
These are just some small changes.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -40,6 +40,7 @@
     <property name="projectHome" location="." />
 
 	<target name="buildjar">
+		<mkdir dir="${projectHome}/bin"/>
```
Comment (TomKimsey) [inline build.xml]: This should be its own commit as its unrelated to the Project name change.

Comment (alexjhawk) [reply build.xml]: fixed
